src/components/EnergyConsumptionChart.tsx

```diff
@@ -1,86 +1,56 @@
 import React from "react"
-import { useRouter } from "next/router"
-import { useTranslation } from "next-i18next"
+import { useEffect, useState } from "react"
 import {
-  Bar,
-  BarChart,
-  Cell,
-  LabelList,
+  BarElement,
+  CategoryScale,
+  Chart as ChartJS,
+  ChartData,
   Legend,
-  ResponsiveContainer,
-  Text,
-  XAxis,
-} from "recharts"
+  LinearScale,
+} from "chart.js"
+import ChartDataLabels from "chartjs-plugin-datalabels"
+import { useRouter } from "next/router"
+import { useTranslation } from "next-i18next"
+import { Bar } from "react-chartjs-2"
 import {
   Box,
   Center,
-  chakra,
+  Text,
   useBreakpointValue,
-  useToken,
+  useColorModeValue,
 } from "@chakra-ui/react"
 
 import type { Lang } from "@/lib/types"
 
+import { wrapLabel } from "@/lib/utils/charts"
 import { isLangRightToLeft } from "@/lib/utils/translations"
 
-type CustomTickProps = {
-  x: number
-  y: number
-  payload: { value: number | string }
-}
-
-type Data = Array<{
-  name: string
-  amount: number
-  color: string
-}>
-
-const RechartText = chakra(Text, {
-  shouldForwardProp: (prop) => {
-    const isValidRechartProp = [
-      "width",
-      "children",
-      "x",
-      "y",
-      "dy",
-      "angle",
-      "scaleToFit",
-      "textAnchor",
-      "verticalAnchor",
-      "breakAll",
-      "maxLines",
-    ].includes(prop)
-
-    return isValidRechartProp
-  },
-})
-
-const CustomTick = ({ x, y, payload }: CustomTickProps) => {
-  return (
-    <g transform={`translate(${x},${y})`}>
-      <RechartText
-        x={0}
-        y={0}
-        dy={15}
-        fill="text"
-        width={50}
-        textAnchor="middle"
-        verticalAnchor="middle"
-        fontSize="2xs"
-      >
-        {payload.value}
-      </RechartText>
-    </g>
-  )
-}
+// ChartDataLabels required to display y-labels on top of bars
+ChartJS.register(
+  CategoryScale,
+  LinearScale,
+  Legend,
+  BarElement,
+  ChartDataLabels
+)
 
 const EnergyConsumptionChart = () => {
   const { t } = useTranslation("page-what-is-ethereum")
-  const textColor = useToken("colors", "text")
   const { locale } = useRouter()
   const isRtl = isLangRightToLeft(locale as Lang)
 
-  const data = useBreakpointValue<Data>({
+  const useIsClient = () => {
```

Comment:
This is a custom hook. It should be defined outside the component (ideally in a separate file in the hooks folder) and used like this
```tsx
const isClient = useIsClient()
```

Reply:
Fixed